kitchen.yml

```diff
@@ -15,15 +15,16 @@ platforms:
   - name: amazonlinux
     driver_config:
       box: mvbcoding/awslinux
-  - name: centos-6
+  - name: amazonlinux-2
   - name: centos-7
```

Comment:
Might want to shove centos 8 in here while you're in the file

Reply:
Done.

Reply:
Converge is failing on Centos 8 - squid itself is starting and working but there's something to do with docker and systemd and squid being dumb that's causing the service start to timeout. It passes in vbox.